include/mqtt/mqtt_esp8266_wifi.h

```diff
@@ -0,0 +1,44 @@
+#ifndef __MQTT_WIFI_H__
+#define __MQTT_WIFI_H__
+
+#ifndef DOMOTICS_DEBUG_LOG_ENABLED
+#define _ESPLOGLEVEL_ 0
+#endif
+
+#include <Arduino.h>
+#include <WiFiEspClient.h>
+
+#include "mqtt.h"
```

Comment:
idem

Reply:
it will link to the mqtt.h in the same folder